transitions/index.js

```diff
@@ -135,20 +144,22 @@ const loadTransitions = (autoEnableSystemTransitions = true) => {
       return logger.warn(`transition ${key} not available.`);
     }
   });
+
+  if (loadError) {
+    logger.error(`Transitions are disabled until the above configuration errors are fixed.`);
```

Comment:
Because this comes *after*:
```js
return logger.warn(`transition ${key} not available.`);
```
I'd be worried that people may think that these warns are part of what they need to fix.